### PR TITLE
Draft of code for SkyCoord parsing from votables

### PR DESCRIPTION
**Pull request by tomdonaldson**
This is a work in progress being discussed at the IVOA interop meeting in Paris - we are exploring ways of getting SkyCoord from VO tables in easy ways.

### CODE REVIEW

File: astropy/coordinates/sky_coordinate.py [hunk 1]
```diff
@@ -1626,26 +1628,49 @@ def guess_from_table(cls, table, **coord_kwargs):
...
+                dec_pm_col = find_column_by_ucd(table, 'pos.pm;pos.eq.dec')
+                if dec_pm_col is not None:
+                    comp_kwargs['pm_dec'] = dec_pm_col
+                
+            # Check VO column metadata first.
+            col = None
+            if use_vo:
+                
+                if comp_name == 'ra':
+                    col = find_column_by_ucd(table, 'POS_EQ_RA_MAIN')
```
Comment (tboch) [inline astropy/coordinates/sky_coordinate.py]: I would also look for pos.eq.ra;meta.main (UCD 1+)

File: astropy/coordinates/sky_coordinate.py [hunk 2]
```diff
@@ -1626,26 +1628,49 @@ def guess_from_table(cls, table, **coord_kwargs):
...
+                elif comp_name == 'dec':
+                    col = find_column_by_ucd(table, 'POS_EQ_DEC_MAIN')
```
Comment (tboch) [inline astropy/coordinates/sky_coordinate.py]: I would also look for pos.eq.dec;meta.main (UCD 1+)

File: astropy/io/votable/ucd.py [hunk 3]
```diff
@@ -192,3 +192,38 @@ def check_ucd(ucd, check_controlled_vocabulary=False, has_colon=False):
...
+    col = find_column_by_ucd(my_table, 'VOX:Image_Title')
+    print ('1st row title value is:', my_table[col.name][0])
+    """
+
+    # Loop through all the columns looking for the UCD
+    for key in table.columns:
+        col = table.columns[key]
+        ucdval = col.meta.get('ucd')
+        if ucdval is not None:
+            if ucd == ucdval:
```
Comment (pllim) [inline astropy/io/votable/ucd.py]: Does user always want only the first column matched? Is there a need for an option to return all matched columns?

File: astropy/coordinates/sky_coordinate.py [hunk 4]
```diff
@@ -1583,7 +1585,7 @@ def radial_velocity_correction(self, kind='barycentric', obstime=None,
 
     # Table interactions
     @classmethod
-    def guess_from_table(cls, table, **coord_kwargs):
+    def guess_from_table(cls, table, use_vo=True, **coord_kwargs):
```
Comment (pllim) [inline astropy/coordinates/sky_coordinate.py]: I am still not sure about having a `use_vo` option here. Is this a slippery slope? In the future, do we have to support `use_fits`, `use_asdf`, or whatever format-specific parsing here?

**Comment by eteq** [comment]
I'm closing this with @tomdonaldson's permission (out-of-band) because we are subsuming it into a larger PR.

**Comment by eteq** [comment]
(for context, #8723 is the larger one)

**Pull request closed without merging**